app/helpers/session_timeout_warning_helper.rb

```diff
@@ -23,10 +23,6 @@ def timeout_refresh_path
   end
 
   def session_modal
-    if user_fully_authenticated?
-      FullySignedInModalPresenter.new(view_context: self, expiration: expires_at)
-    else
-      PartiallySignedInModalPresenter.new(view_context: self, expiration: expires_at)
-    end
+    SessionTimeoutModalPresenter.new(user_fully_authenticated: user_fully_authenticated?)
```

Comment:
we ditched passing in `expires_at` and ended up hardcoding `Time.zone.now` in the partial, did we intend to keep passing it in?

Reply:
> we ditched passing in `expires_at` and ended up hardcoding `Time.zone.now` in the partial, did we intend to keep passing it in?

Nah, as mentioned in the original comment, the initial time remaining we assign would never actually be seen be a user in real-world usage, since the JavaScript will dynamically update the expiration (and consequently the text) by the time it's made visible.